{{cookiecutter.project_dashed}}-src/{{cookiecutter.project_namespace}}/config.py

```diff
@@ -34,9 +43,14 @@ class TestProfile(object):
     KEG_KEYRING_ENABLE = False
 
     # Mail related tests need to have this set, even though actual email is not generated.
-    MAIL_DEFAULT_SENDER = '{{cookiecutter.developer_email}}'
+    MAIL_DEFAULT_SENDER = 'test@test.com'
 
     CELERY = {
```

Comment:
Not using Celery config function

Reply:
I am not sure I understand this, what do you mean?

Reply:
I see what you mean now... fixed.